modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceAbstractIntegrationTest.java

```diff
@@ -342,6 +342,34 @@ public HttpResponse sendLoginPost(HttpClient client, String sessionDataKey)
 		return response;
 	}
 
+	/**
+	 * Send login post request with given username and password credentials.
+	 *
+	 * @param client
+	 *            - Http client
+	 * @param sessionDataKey
+	 *            - Session data key
+	 * @param username
+	 *            - Username
+	 * @param password
+	 *            - Password
+	 * @return http response
+	 * @throws ClientProtocolException
+	 * @throws IOException
+	 */
+	public HttpResponse sendLoginPostForCustomUsers(HttpClient client, String sessionDataKey, String username, String password)
+			throws ClientProtocolException,
+			IOException {
+		List<NameValuePair> urlParameters = new ArrayList<>();
```

Comment:
Add a new line after the method signature. Check other places as well.

Reply:
Added new lines after the method signatures